packages/augur-api/src/api/Events.ts

```diff
@@ -23,9 +21,9 @@ export class Events {
         return this.parseLogs(logs);
     }
 
-    public getEventTopics(eventName: string) {
+    public getEventTopics = (eventName: string) => {
```

Comment:
@robert-e-davidson3 This is super weird.

Any idea why this fixes it? @adrake33 any thoughts?